<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smiley</class>
 <widget class="QDialog" name="smiley">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smiley</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="icon1">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="icon1" resource="icons.qrc">
        <normaloff>:/icons/smiley/icons/icon1.png</normaloff>:/icons/smiley/icons/icon1.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="icon2">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/smiley/icons/icon2.png</normaloff>:/icons/smiley/icons/icon2.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="icon3">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/smiley/icons/icon3.png</normaloff>:/icons/smiley/icons/icon3.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="icon4">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/smiley/icons/icon4.png</normaloff>:/icons/smiley/icons/icon4.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="icon5">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/smiley/icons/icon5.png</normaloff>:/icons/smiley/icons/icon5.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="icon6">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/smiley/icons/icon6.png</normaloff>:/icons/smiley/icons/icon6.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>smiley</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>smiley</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
